management/server/sql_store.go

```diff
@@ -450,7 +450,7 @@ func (s *SqlStore) SaveUsers(accountID string, users map[string]*User) error {
 func (s *SqlStore) SaveUser(ctx context.Context, lockStrength LockingStrength, user *User) error {
 	startTime := time.Now()
 
-	result := s.db.WithContext(ctx).Clauses(clause.Locking{Strength: string(lockStrength)}).Save(user)
+	result := s.db.Clauses(clause.Locking{Strength: string(lockStrength)}).Save(user)
 	if result.Error != nil {
 		if errors.Is(result.Error, context.Canceled) {
 			return status.NewStoreContextCanceledError(time.Since(startTime))
```

Comment:
Since we're no longer passing context into db calls, the context cancellation check is not needed anymore. GORM won’t report context cancellation without the context, so it's better to remove these checks

Reply:
You are right. I will remove it